modAionImpl/test/org/aion/zero/impl/db/AionRepositoryImplTest.java

```diff
@@ -371,4 +375,50 @@ public void testGetSnapshotToRoot() {
         assertThat(repository.getBalance(account2)).isLessThan(snapshot.getBalance(account2));
         assertThat(repository.getBalance(account3)).isLessThan(snapshot.getBalance(account3));
     }
+
+    @Test
+    public void testImportTrieNode() {
+        AionRepositoryImpl repository = AionRepositoryImpl.createForTesting(repoConfig);
+        IByteArrayKeyValueDatabase db = repository.getStateDatabase();
+        byte[] nodeKey = RequestTrieDataTest.nodeKey;
+        byte[] altNodeKey = RequestTrieDataTest.altNodeKey;
+        byte[] smallNodeKey = RequestTrieDataTest.smallNodeKey;
+        byte[] leafValue = ResponseTrieDataTest.leafValue;
+        byte[] branchValue = ResponseTrieDataTest.branchValue;
```

Comment:
Might be safer to just duplicate those values here instead of referring to them (I think if someone is updating ResponseTriedatatest.branchValue they might not expect that some other test also uses it)

Reply:
Done.